### PR TITLE
Use Kontena::Command hooks and new authentication mechanism.

### PR DESCRIPTION
**Pull request by kke**

### CODE REVIEW

File: lib/kontena/machine/packet/cloudinit_master.yml [hunk 1]
```diff
@@ -88,8 +88,11 @@ coreos:
             --link kontena-server-mongo:mongodb \
             -e MONGODB_URI=mongodb://mongodb:27017/kontena_server \
 <% end -%>
-<% if auth_server %>
-            -e AUTH_API_URL=<%= auth_server %> \
+<% if server_name %>
```
Comment (jakolehm) [inline lib/kontena/machine/packet/cloudinit_master.yml]: No newlines allowed

File: lib/kontena/machine/packet/cloudinit_master.yml [hunk 2]
```diff
@@ -88,8 +88,11 @@ coreos:
...
+            -e KONTENA_SERVER_NAME=<%= server_name %> \
```
Comment (jakolehm) [inline lib/kontena/machine/packet/cloudinit_master.yml]: `KONTENA_SERVER_NAME` => `SERVER_NAME`

File: lib/kontena/machine/packet/cloudinit_master.yml [hunk 3]
```diff
@@ -88,8 +88,11 @@ coreos:
...
+<% end -%>
+<% if initial_admin_code %>
+            -e KONTENA_INITIAL_ADMIN_CODE=<%= initial_admin_code %> \
```
Comment (jakolehm) [inline lib/kontena/machine/packet/cloudinit_master.yml]: `KONTENA_INITIAL_ADMIN_CODE` => `INITIAL_ADMIN_CODE`

**Comment by kke** [comment]
Fixes applied

**Comment by jakolehm** [comment]
Needs rebase

**Comment by kke** [comment]
Rebased

**Pull request merged**